libseccomp/src/filter_context.rs

```diff
@@ -166,10 +166,9 @@ impl ScmpFilterContext {
     /// ctx.add_arch(ScmpArch::X86)?;
     /// # Ok::<(), Box<dyn std::error::Error>>(())
     /// ```
-    pub fn add_arch(&mut self, arch: ScmpArch) -> Result<bool> {
+    pub fn add_arch(&mut self, arch: ScmpArch) -> Result<()> {
         match unsafe { seccomp_arch_add(self.ctx.as_ptr(), arch.to_sys()) } {
-            0 => Ok(true),
-            MINUS_EEXIST => Ok(false),
```

Comment:
Could you add the following comment?

```rust
// The libseccomp returns -EEXIST if the specified architecture is already
// present. Succeed silently in this case, as it's not fatal, and the
// architecture is present already.
```

Besides, please remove the following doc commentes.
https://github.com/libseccomp-rs/libseccomp-rs/pull/218/files#diff-cf472dd53e08212cd16fb7c9fa788b407cae006cb94ed5b7249850c6026618edR145-R147